### PR TITLE
New package: ssh-tools-1.6

### PR DESCRIPTION
**Pull request by vaporup**
#### General

- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?

- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 

- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

### CODE REVIEW

File: srcpkgs/ssh-tools/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'ssh-tools'
+pkgname=ssh-tools
+version=1.6
+revision=1
+makedepends="help2man"
```
Comment (abenson) [inline srcpkgs/ssh-tools/template:5]: hostmakedepends

Comment (vaporup) [reply srcpkgs/ssh-tools/template:5]: adding hostmakedepends or replacing makedepends with hostmakedepends?

Comment (ericonr) [reply srcpkgs/ssh-tools/template:5]: help2man should be hostmakedepends

Comment (Chocimier) [reply srcpkgs/ssh-tools/template:5]: help2man is a problem for cross build, we usually omit it everywhere, as help is already there

Comment (vaporup) [reply srcpkgs/ssh-tools/template:5]: So I can drop makedepends and the manpage build steps completely?

Comment (ericonr) [reply srcpkgs/ssh-tools/template:5]: Since you are calling it manually in this case, definitely.

Comment (vaporup) [reply srcpkgs/ssh-tools/template]: done

**Comment by vaporup** [comment]
New upstream release. But since this is still not merged and I have no idea why, closing it now.

**Pull request closed without merging**